specs/DesignDiscussions/InputLabelsOptionLabel.md

```diff
@@ -0,0 +1,440 @@
+	"version": "1.0",
+	"body": [
+		{
+			"type": "Input.Text",
+			"id": "nameInput",
+			"label": "Please enter your name:",
+			"placeholder": "Last, First"
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "OK"
+		}
+	]
+}
+```
+
+## Formatting
+The above approach provides a simple means by which to specify the label, but doesn't immediately provide the ability to format the label. Card authors today who are using TextBlocks or RichTextBlocks as labels have the ability to set any of our formatting properties to control the size, color, font weight, etc. of their text. The schema proposed above does not give them that flexibility. One option to address this would be to allow card authors to specify a TextBlock or a RichTextBlock as a label in addition to supporting a simple string.
+
+There is another concern, however, which must be balanced against the desire to provide card authors with the ability to format their labels. Hosts may have label formatting in their app which they want to apply consistently to cards they are displaying. This applies particularly to how they mark fields as required or optional (per spec at [InputValidation.md](https://github.com/microsoft/AdaptiveCards/blob/master/specs/DesignDiscussions/InputValidation.md)). While by default we will mark required inputs with a `*`, we may want to allow hosts more control. Perhaps some apps wish to show labels for required inputs in bold or italics. If we allows hosts to specify formatting based on what is required, however, and we allow authors to specify formatting via a TextBlock or RichTextBlock, we can easily run into cases where the formatting comes into conflict.
+
+This conflict speaks to a general tension inherent in the design of Adaptive Cards. There's always going to be a trade off between giving the author more power, and letting the host control the experience for their users. We need to determine what balance we want to strike in this case.
+
+In order to gather more data to help us balance the needs of the card authors and hosts, we sent a survey to customers who have expressed an interest in improving out input story. The following options are under consideration, and will be evaluated against survey data to determine which if any will be included in our v1 implementation.
+
+The survey sent included 10 questions including the topics of input validation and input labels, for this spec we will only review the 4 questions that focus on the former topic. The results for this questions were used to leverage on some of the decisions taken on this spec document and can be found in the [Appendix](Appendix).
+
+> ### Default formatting for labels
+>
+> Reviewing the results from Q8 of the survey, we can see that most voters usually have a good experience with the default rendering of a TextBlock for rendering a label. From this result we can decide that string label rendering should be performed as if we were rendering a default TextBlock.
```

Comment:
Great.. so let's just specify that upfront then? That a label for now will just be a simple string value @almedina-ms ? (@paulcam206 what was your suggested approach again for when we choose to enhance label support to be richer than plain strings?)

Reply:
One possibility is that we can ship v1 where the `label` is specified as a simple string. In subsequent releases we can support `label` being an object (i.e. an `AdaptiveElement`, or the equivalent):

v1:
``` json
{
    "type": "Input.Text",
    "label": "First name"
}
```

v2 (could use container or columnset... whatever):
``` json
{
   "type": "Input.Text",
    "label": {
        "type": "RichTextBlock",
        "inlines": [
            "some sample text",
            "maybe something with a glyph here!"
        ]
    }
}
```

Reply:
Thanks @Paulcam206; i think that's precisely the direction we ended up going last week where subsequently, per @matthidinger's follow up proposal, v2's type would potentially be a separate "Label" type..